common/common.go

```diff
@@ -31,6 +31,7 @@ const (
 	MetadataServiceIP      = "169.254.169.255"
 	MetadataServicePubPort = 80
 	MetadataServicePrvPort = 2375
+	MetadataServiceRegSock = "/var/run/rkt-metadata-svc.sock"
```

Comment:
The modern layout uses /run and /var/run is a symlink to /run: http://www.freedesktop.org/software/systemd/man/file-hierarchy.html#/var/run

So I would stick with /run unless you wanted compatibility with the old layout but considering rocket needs a modern kernel that seems unnecessary.